tests/Install-DbaMultiTool.second.Tests.ps1

```diff
@@ -0,0 +1,36 @@
+$CommandName = $MyInvocation.MyCommand.Name.Replace(".second.Tests.ps1", "")
```

Comment:
Please put all tests into the same file.

Reply:
That was my original approach, but doing so results in [this error](https://ci.appveyor.com/project/sqlcollaborative/dbatools/builds/36324572/job/yt1g6extt9mjtha1), that I am not sure the cause behind. 

Using the [second file](https://github.com/sqlcollaborative/dbatools/blob/development/contributing.md#appveyor-setup) approach (from contributing.md) results in that test succeeding. 

It is a minor addition to the existing tests for similar commands, so can also just be removed altogether potentially as well.

Reply:
Your test only needs to run against `$script:instance3`. I do not see any reason to have it run on other versions.